gstd/src/prog/mod.rs

```diff
@@ -18,6 +18,8 @@
 
 //! Program creation module
 
+pub mod program_creator;
```

Comment:
I see no tests for that. The main aim of the test is to guarantee, that no repeated program ids will occur.